README.md

```diff
@@ -36,90 +40,51 @@ The Nulecule specification enables complex applications to be defined, packaged
 
 ## Deployment User Experience
 
-The Nulecule specification has been implemented in the [Atomic App reference implementation](https://github.com/projectatomic/atomicapp).  Atomic App currently supports docker containers and kubernetes and docker orchestration providers.  The [atomic command](https://github.com/projectatomic/atomic) is used to run the container that contains the Nulecule specification and the Atomic App implementation.
-
-This example is a single container application based on the centos/httpd image, but you can use your own.
-
-You may wish to run the Nulecule from an empty directory as it will copy the Nulecule files to the working directory for inspection every time it is run.
-
-### Option 1: Non-interactive defaults
-
-Run the image. It will automatically use kubernetes as the orchestration provider.  This will become interactive and prompt for defaults if the Nulecule file doesn't provide defaults for all of the parameters.
-
-```
-[sudo] atomic run projectatomic/helloapache
-```
-
-### Option 2: Unattended
-
-1. Create the file `answers.conf` with these contents:
-
-    This sets up the values for the two configurable parameters (image and hostport) and indicates that kubernetes should be the orchestration provider.
-
-        [general]
-        provider = kubernetes
-
-        [helloapache-app]
-        image = centos/httpd # optional: choose a different image
-        hostport = 80        # optional: choose a different port to expose
-1. Run the application from the current working directory
-
-        $ [sudo] atomic run projectatomic/helloapache
-        ...
-        helloapache
+The Nulecule specification has been implemented in the [Atomic App reference implementation](https://github.com/projectatomic/atomicapp).  Atomic App currently supports docker as well as container orchestrators such as Kubernetes, OpenShift and Marathon.  The [atomic](https://github.com/projectatomic/atomic) or [atomicapp](https://github.com/projectatomic/atomicapp] command is used to run the container that contains the Nulecule specification.
```

Comment:
Let's not call atomic app a "reference" implementation. Let's just refer to it as an implementation